### PR TITLE
Login Page Changes

### PR DESCRIPTION
**Pull request by Saravanagithub**

### CODE REVIEW

File: src/main/java/com/paydesk/controller/HelloWorldController.java [hunk 1]
```diff
@@ -19,4 +20,12 @@ public ModelAndView showMessage(
 		mv.addObject("name", name);
 		return mv;
 	}
+
```
Comment (ghost) [inline src/main/java/com/paydesk/controller/HelloWorldController.java:22]: We need to rename this class. From helloWorld Controller to LoginController

File: src/main/webapp/WEB-INF/dispatcher-servlet.xml [hunk 2]
```diff
@@ -19,5 +23,11 @@ http://www.springframework.org/schema/context/spring-context-3.0.xsd">
 		</property>
 	</bean>
 	
+	 <mvc:resources mapping="/resources/**" location="/resources/theme1/css/" cache-period="31556926"/>
```
Comment (ghost) [inline src/main/webapp/WEB-INF/dispatcher-servlet.xml]: Can we remove theme1. lets have it directly in resources/css

Comment (sprassanna) [inline src/main/webapp/WEB-INF/dispatcher-servlet.xml]: What's the use of cache-period="31556926" in this

File: src/main/webapp/WEB-INF/views/login.jsp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  <div class="form">
+    <h2>Login to your account</h2>
+    <form>
+      <input type="text" placeholder="Username"/>
+      <input type="password" placeholder="Password"/>
+      <button>Login</button>
+    </form>
+  </div>
+  <div class="form">
+    <h2>Create an account</h2>
```
Comment (sprassanna) [inline src/main/webapp/WEB-INF/views/login.jsp:41]: do we have sign up UI as part of Login Page.

File: src/main/java/com/paydesk/controller/HelloWorldController.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package com.paydesk.controller;
 
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
```
Comment (sprassanna) [inline src/main/java/com/paydesk/controller/HelloWorldController.java]: Removed this unused import

File: src/main/webapp/WEB-INF/views/login.jsp [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    <form>
+      <input type="text" placeholder="Username"/>
+      <input type="password" placeholder="Password"/>
+      <input type="email" placeholder="Email Address"/>
+      <input type="tel" placeholder="Phone Number"/>
+      <button>Register</button>
+    </form>
+  </div>
+  <div class="cta"><a href="http://andytran.me">Forgot your password?</a></div>
```
Comment (sprassanna) [inline src/main/webapp/WEB-INF/views/login.jsp]: Its pointing to different URL, remove that

File: src/main/webapp/resources/theme1/css/style.css [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+}
+.pen-title h1 {
+  margin: 0 0 20px;
+  font-size: 48px;
+  font-weight: 300;
+}
+.pen-title span {
+  font-size: 12px;
+}
+.pen-title span .fa {
```
Comment (sprassanna) [inline src/main/webapp/resources/theme1/css/style.css]: Where .pen-title span ; .pen-title span .fa and .pen-title span element are defined in application . I can't find that

**Comment by sprassanna** [comment]
Need code walk through on css

File: src/main/webapp/WEB-INF/views/login.jsp (hunk 3 above)
Comment (Saravanagithub) [inline src/main/webapp/WEB-INF/views/login.jsp:41]: We have it but currently we are hiding it

File: src/main/webapp/resources/theme1/css/style.css (hunk 6 above)
Comment (Saravanagithub) [inline src/main/webapp/resources/theme1/css/style.css]: these csss are defined for sign up, which is currently hidden

File: src/main/webapp/WEB-INF/dispatcher-servlet.xml (hunk 2 above)
Comment (Saravanagithub) [inline src/main/webapp/WEB-INF/dispatcher-servlet.xml]: it caches the resources like css, jss and other resources

**Comment by Saravanagithub** [comment]
Please continue review

**Comment by ghost** [comment]
Looks Good :+1: . I think the other changes we could do in our next sprint. Cleaning up and all

**Comment by sprassanna** [comment]
Looks Good :+1:

**Comment by MithileshAnna** [comment]
Looks good.

**Pull request merged**